### PR TITLE
3.0 beta

### PR DESCRIPTION
**Pull request by sqs**
This is the announcement blog post for Sourcegraph 3.0 beta.

- [**View rendered draft announcement**](https://github.com/sourcegraph/about/blob/3.0-beta/blogposts/announcing-sourcegraph-3.0-beta.md)

### CODE REVIEW

**Comment by sqs** [comment]
Yes, I am completely rewriting it now.

**Comment by sqs** [comment]
OK, you can review it now. It's still WIP but any sections that have text are ready for review.

File: blogposts/announcing-sourcegraph-3.0-beta.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Sourcegraph 3.0-beta available
+
+The beta release of Sourcegraph 3.0 is now available (as version `3.0.0-beta`). Sourcegraph is an open-source, self-hosted code search and browsing tool for teams, with an efficient web interface and feels-like-native integration into your code host. This release comes with several big core updates:
```
Comment (slimsag) [inline blogposts/announcing-sourcegraph-3.0-beta.md]:
```suggestion
The beta release of Sourcegraph 3.0 is now available (as version tag `3.0.0-beta`). Sourcegraph is an open-source, self-hosted code search and browsing tool for teams, with an efficient web interface and feels-like-native integration into your code host. This release comes with several big core updates:
```

**Comment by slimsag** [comment]
@sqs Where are you thinking mgmt console info will fit into this doc?

**Comment by sqs** [comment]
@slimsag 	I need to add a section for it (will do now).

**Comment by sqs** [comment]
@slimsag 	Added the mgmt console bullet list item and section

**Comment by dadlerj** [comment]
We should add a mention of our pricing changes (https://github.com/sourcegraph/about/pull/70). Do you agree?

**Comment by nicksnyder** [comment]
3.0.0-beta is tagged. Can you update the changelog with this PR too before merging?

**Comment by dadlerj** [comment]
@nicksnyder I think the changelog is in sourcegraph/sourcegraph, no?

**Comment by dadlerj** [comment]
Hmm looks like the link on https://sourcegraph.com/start leads to a 404: https://about.sourcegraph.com/changelog

**Pull request merged**